_posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Edge Example App: Sense the Internet Part 2"
+date:   2022-09-05 03:27:05 +0100
+categories: flotta
+author: Ahmad Ateya <ahmad.m.ateya@gmail.com>
+tags:
+- flotta
+- devices
+- workloads
+- gsoc
+
+summary-1:
+---
+Edge Example App is an app for Flotta Edge devices, with a workload that will be deployed on the device that has two main features:
+- Sensing the Internet (which helps to construct devices network topology). <= this article (Part 2)
+- Read CPU temperature (which indicate how much load the device is handling).
+
+## How this app works
+This app is the presenting layer of the Flotta Edge devices, get its data from the backend app endpoints and display it to the user.
```

Comment:
WDYT to add link to the backend app repo?

Reply:
good idea, added it